settings/eslint/eslintrc.js

```diff
@@ -0,0 +1,66 @@
+const commonParserOptions = {
+  ecmaFeatures: {
```

Comment:
Are these options needed? I've never seen them defined without losing any linting

Reply:
Seems to work without `ecmaFeatures`